CMakeLists.txt

```diff
@@ -47,12 +47,18 @@ message(STATUS "build type is ${CMAKE_BUILD_TYPE}")
 
 set_property(CACHE CMAKE_BUILD_TYPE PROPERTY STRINGS "Debug;Release;RelWithDebInfo;MinSizeRel;Coverage")
 
-execute_process(
-	COMMAND git describe --always --tags
-	OUTPUT_VARIABLE git_tag
-	OUTPUT_STRIP_TRAILING_WHITESPACE
-	WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
-	)
+if(EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/.git)
+        execute_process(
+                COMMAND git describe --always --tags
+                OUTPUT_VARIABLE git_tag
+                OUTPUT_STRIP_TRAILING_WHITESPACE
+                WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
+                )
+elseif($ENV{ECL_LIB_GIT_TAG_STR})
+        set(git_tag $ENV{ECL_LIB_GIT_TAG_STR})
+else()
+        set(git_tag "")
```

Comment:
Any use for keeping this case or should it be a FATAL_ERROR?

Reply:
I simplified these cases out, as it looks like ${git_tag} is only used for a [STATUS message](https://github.com/PX4/ecl/pull/850/files#diff-af3b638bc2a3e6c650974192a53c7291L57)